app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -218,6 +204,9 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
 
         HintBinder.bindHintsToView(hintViewModel, hintBarContainer, animate = true)
                 .forEach { startStopCompositeDisposable.add(it) }
+
+        cursorView.setup(context!!.serviceLocator.cursorController)
```

Comment:
The `cursorView` is owned by the `WebRenderFragment`: why aren't the key events also handled in the `webRenderFragment`?